### DISCUSSION TITLE
Microsoft  Support

### DISCUSSION THREAD

**Discussion by shaharphilips** [CoreWCF/CoreWCF#1504] [General]
Hi,

We are evaluating the use of CoreWCF to migrate our application to .NET. Is there any up-to-date support notice from Microsoft for this awsome product? Having MS support is a very significant factor in the decision to move forward with the project.
The current support notice seems to be quite old - https://github.com/CoreWCF/CoreWCF/blob/main/Documentation/Microsoft-Support.md

Shahar

**Comment by mconnew** [comment]
That document was a temporary placeholder until the official Microsoft documentation was in place. Please see https://dotnet.microsoft.com/en-us/platform/support/policy/corewcf for an up to date support statement.
